ui/components/MeshAdapterConfigComponent.js

```diff
@@ -276,7 +277,7 @@ const MeshAdapterConfigComponent = (props) => {
     );
   };
 
-  const handleClick = (adapterLoc) => () => {
```

Comment:
Thank you for fixing this and making them normal functions rather than HOF functions, can you also make sure that wherever they're getting called is right and properly working

Reply:
for other functions too

Reply:
I have checked it already, they were working fine. But still I will check again and will inform you soon.